app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/previous_return_exists_with_same_completion_date.scala.html

```diff
@@ -39,12 +39,14 @@
     previousReturn.propertyAddress match {
         case Address.UkAddress(line1, _, _, _, _) => line1
         case Address.NonUkAddress(line1, _, _, _, _, _) => line1
+        case _ => ""
     }
 }
 
 @line2Display = @{
     previousReturn.propertyAddress match {
         case Address.UkAddress(_, _, _, Some(town), postcode) => s"$town, ${postcode.value}"
+        case Address.NonUkAddress(_, Some(area), Some(townOrCity), _, _, _) => s"$area, $townOrCity"
```

Comment:
there is no other case below this line either

Reply:
There's a case where it's a NonUKAddress and the optional area/townorcity are None

Reply:
oh yeah,,,what happens if the's a uk address with no town then? Do we still want to display the postcode?